plugins/modules/jcliff.py

```diff
@@ -376,6 +376,29 @@
             type: str
             required: False
 
+      jms:
+        description:
+          - jms-queue.
+        type: list
+        suboptions:
+
+          queue_address:
+            description:
+              - Name of the queue.
+            type: str
```

Comment:
@sabre1041 So here  are my issues:

1) This approach (passing the entries as str) never worked for me (locally and within a docker container). I suspect that like the issue we had with the demo, some other will run into problem when they try this feature.

2) The approach is a bit hacky: we are passing an array as a string that then later get iterated over, as an array. I think this is going to get us in trouble with old/newer version of the templating (or simply if the value provided uses variables or is fully enclosed into double quotes.

3) Bottom line, this should be a list of strings, so that we are type safe (as much as possible in this constraints) and we can implement it similarly with Artemis when it comes to that.